karamel-ui/deploy-packages-server.sh

```diff
@@ -29,9 +27,7 @@ cd karamel-ui/target
 cp -r appassembler/* $dist/
 cp ../README.linux $dist/README.txt
 mkdir $dist/examples
-cd ../../../karamel-examples
-git checkout-index -a -f --prefix=../karamel/karamel-ui/target/$dist/examples/
-cd ../karamel/karamel-ui/target
+cp ../../../karamel-chef/cluster-defns/* $dist/examples/
```

Comment:
don't copy all of the cluster defns. most of them won't work.
Have a naming convention for those we want to include in karamel. Currently, in karamel-chef, the naming convention is 1.name-blah.yml or 3.name-blah.yml
If we have
hopsworks-*.yml
as the official yml files, that will be fine.
We should have 2 examples each for AWS, GCP, baremetal- single node, and 3-node clusters. That will be enough to begin with.

Reply:
cp ../../../karamel-chef/cluster-defns/hopsworks*.yml $dist/examples/

Reply:
I added 4 more cluster-defns in karamel chef for AWS and GCP installations.
https://github.com/logicalclocks/karamel-chef/pull/77/files
The 1 and 3-node should be the same as what we have and it should just work, no need to add more.
https://github.com/logicalclocks/karamel-chef/tree/master/cluster-defns